runtime/lua/vim/lsp/util.lua

```diff
@@ -1272,6 +1272,23 @@ function M.character_offset(buf, row, col)
   return str_utfindex(line, col)
 end
 
+function M.format_formatexpr()
```

Comment:
let's change this just to `formatexpr` since the name is a bit redundant currently.